esx_service/utils/vmdk_utils.py

```diff
@@ -73,10 +73,9 @@ def init_datastoreCache(force=False):
     from local ESX host. force=True will force it to ignore current cache
     and force init
     """
-    logging.debug("init_datastoreCache:  %s", datastores)
-
```

Comment:
Nit: I'd suggest to add logs before and after acquiring locks so that it's easier to debug potential dead lock issues.